.github/workflows/main.yml

```diff
@@ -97,3 +72,128 @@ jobs:
             -m "Build results of ${GITHUB_REF#refs/heads/*} kaitai-io/ci_targets@$GITHUB_SHA" \
             -b "$TARGET$SUBTARGET/$VARIETY" \
             -- --exclude=.git --exclude=.travis.yml tests/test_out
+
+  dockerized_test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tsv:
+          - target: cpp_stl
+            subtarget: _98
+            implementation: gcc4.8
+          - target: cpp_stl
+            subtarget: _11
+            implementation: gcc4.8
+          - target: cpp_stl
+            subtarget: _98
+            implementation: gcc11
+#          - target: cpp_stl
+#            subtarget: _11
+#            implementation: gcc11
+          - target: java
+            subtarget: ''
```

Comment:
It seems that it's actually possible to omit these empty string settings, which would shorten the config a bit if we don't need any meaningful subtarget for these jobs. If you see some value in keeping them, feel free to - I'm just noting they're not necessary.

Reply:
Very good point, thanks! Let me try that.